src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vedtak/sammensattkontrollsak/SammensattKontrollsakDtoMapper.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.ks.sak.kjerne.behandling.steg.vedtak.sammensattkontrollsak
+
+import no.nav.familie.ks.sak.api.dto.OpprettSammensattKontrollsakDto
+import no.nav.familie.ks.sak.api.dto.SammensattKontrollsakDto
+import org.springframework.stereotype.Component
+
+@Component
+class SammensattKontrollsakDtoMapper {
```

Comment:
Litt usikker om vi trenger en @Component  mapper for det her.
Kan kanskje bare være greit å ha en extension function som kan gjøre dette for oss, på samme måte som i BA?

Hvis man velger å gå for mapper varianten så bør denne flyttes til mapper mappen i api pakken.

Reply:
Har man en extension function som også takler null, så slipper man denne if sjekken i controlleren:
```
        return if (sammensattKontrollsak != null) {
            ResponseEntity.ok(Ressurs.success(sammensattKontrollsakDtoMapper.map(sammensattKontrollsak)))
        } else {
            ResponseEntity.ok(Ressurs.success(null))
        }
```